File: cfgs/pythoncfgs/MINIMUM_ROTATIONS_REQUIRED_GET_STRING.py
Nodes
1;statement;foo_1 = foo_2 + foo_3;8;
2;statement;foo_1 = fun_1(foo_2);9;
3;conditional;foo < foo;10;
4;statement;return foo_1;14;
5;statement;foo_1 = foo_2[foo_3:foo_4 + foo_5];11;
6;conditional;foo_1 == foo_2;12;
7;statement;return foo_1;13;
8;exit;;
Edges
0;2;
1;3;
2;4;5;
3;8;
4;6;
5;3;7;
6;8;
7;

File: cfgs/java_cfgs_foo/MINIMUM_ROTATIONS_REQUIRED_GET_STRING.java
Nodes
1;special;;;
2;statement;String tmp = foo_1 + foo_2;14;
3;statement;int n = fun_1();15;
4;statement;int i = 1;16;
5;conditional;foo_1 <= foo_2;17;
6;statement;String substring = fun_1(foo_2, fun_2());19;
7;conditional;foo_1 == foo_2;20;
8;statement;return foo_1;20;
9;exit;;;
10;statement;i++;18;
11;statement;return foo_1;22;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;11;
5;7;
6;8;10;
7;9;
8;
9;5;
10;9;
